modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/Parameter.java

```diff
@@ -113,12 +114,12 @@
    * @return whether or not this parameter is hidden
    */
   boolean hidden() default false;
-  
+
   /**
-   * Provides an array examples of the schema.  When associated with a specific media type, the example string shall be parsed by the consumer to be treated as an object or an array.  Ignored if the properties content or array are specified.
-   * @return the list of examples for this parameter
+   * An array of examples used to show the use of the associated schema.
+   * @return the list of examples
```

Comment:
change to @return array of examples of the parameter

Reply:
Changed!